glotaran/builtin/file_formats/sdt/test/test_file_readers.py

```diff
@@ -17,8 +17,8 @@ def test_read_sdt(test_file_path, result_file_path, index):
                             dtype={"Delay": np.float, "Data": np.uint16})
     result_df.Delay = result_df.Delay * 1e-9
     result_traces = pd.DataFrame([result_df.Data.values], columns=result_df.Delay)
-    assert isinstance(test_dataset, xr.Dataset)
```

Comment:
I am unsure if xr.Dataset can be replaced with cr.DataArray without unintended consequences

Reply:
It can, you can always extract the things you want to fit as array with your favorite tool. The sdt file reader will suffer a little from that but nothing to busy. In the end, outside of `nc` a file represents 1 array, so the whole file abstraction is now centered around that a file represents one array.